src/strava_py/cli.py

```diff
@@ -23,6 +30,10 @@ def main():
     plot_facets(df, output_file=args.output_file)
     print(f"Saved to {args.output_file}")
 
+    print("Plotting map...")
+    plot_facets(df, output_file=args.output_file)
+    print(f"Saved to {args.output_file}")
```

Comment:
Call `plot_map` and also pass the CLI args:
```suggestion
    plot_map(
        df,
        lon_min=args.lon_min,
        lon_max=args.lon_max,
        lat_min=args.lat_min,
        lat_max=args.lat_max,
        alpha=args.alpha,
        linewidth=args.linewidth,
        output_file=args.output_file,
    )
    print(f"Saved to {args.output_file}")
```

Reply:
We're also re-using `args.output_file` and therefore writing over the facets image.

Some options:

* Remove the argument entirely
* Add a second one for the map plot
* Change the argument to be a prefix instead (e.g. `strava`), and we append `-facets.png` and `-map.png`

I think the third, as there may be more visualisations added soon, and it will be useful to be able to have different names for different runs (for example, I might generate output for each year of activities, or summer vs. winter).